### PR TITLE
zh-tw: sync with english version for Array.of()

### PR DESCRIPTION
**Pull request by jchuder**
Sync with english version.

### CODE REVIEW

File: files/zh-tw/web/javascript/reference/global_objects/array/of/index.md [hunk 1]
```diff
@@ -66,6 +86,9 @@ if (!Array.of) {
 
 ## 參見
 
+- [Polyfill of `Array.of` in `core-js`](https://github.com/zloirock/core-js#ecmascript-array)
 - {{jsxref("Array")}}
 - {{jsxref("Array.from()")}}
 - {{jsxref("TypedArray.of()")}}
+- [`Array.of()` & `Array.from()` proposal](https://gist.github.com/rwaldron/1074126)
+- [`Array.of()` polyfill](https://gist.github.com/rwaldron/3186576)
```
Comment (yin1999) [inline files/zh-tw/web/javascript/reference/global_objects/array/of/index.md:94]: Removed in mdn/content#25802

```suggestion
```

File: files/zh-tw/web/javascript/reference/global_objects/array/of/index.md [hunk 2]
```diff
@@ -38,12 +34,36 @@ Array.of(element0[, element1[, ...[, elementN]]])
...
-Array.of(1);         // [1]
-Array.of(1, 2, 3);   // [1, 2, 3]
+Array.of(1); // [1]
+Array.of(1, 2, 3); // [1, 2, 3]
 Array.of(undefined); // [undefined]
 ```
 
+### 在非陣列建構中使用 of()
+
+The `of()` method can be called on any constructor function that accepts a single argument representing the length of the new array.
```
Comment (yin1999) [inline files/zh-tw/web/javascript/reference/global_objects/array/of/index.md:47]: 请翻译一下这句话

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-tw/web/javascript/reference/global_objects/array/of/index.md (hunk 2 above)
Comment (yin1999) [reply files/zh-tw/web/javascript/reference/global_objects/array/of/index.md:47]:
```suggestion
可以在接受表示新陣列長度的單個參數的任何建構式上調用 `of()` 方法。
```

Comment (jessiehu907) [reply files/zh-tw/web/javascript/reference/global_objects/array/of/index.md:47]: 抱歉, 沒有即時看到, tks!

File: files/zh-tw/web/javascript/reference/global_objects/array/of/index.md [hunk 3]
```diff
@@ -66,6 +86,7 @@ if (!Array.of) {
 
 ## 參見
 
+- [Polyfill of `Array.of` in `core-js`](https://github.com/zloirock/core-js#ecmascript-array)
 - {{jsxref("Array")}}
```
Comment (yin1999) [inline files/zh-tw/web/javascript/reference/global_objects/array/of/index.md:90]:
```suggestion
- [`Array()`](/zh-TW/docs/Web/JavaScript/Reference/Global_Objects/Array/Array)
```

**Review by yin1999 — APPROVED** [review]
Thanks

**Comment by schalkneethling** [comment]
Welcome to the community, @jessiehu907  🎉 Thank you for your contribution. If you are interested in more ways to contribute to MDN Web Docs, please have a look at our [contributor task board](https://github.com/orgs/mdn/projects/25). If you have any questions or need help finding something to work on, please ask for help in our [community discussions](https://github.com/orgs/mdn/discussions). We look forward to more of your contributions. 🙌

**Pull request merged**